apps/sms/js/dialog.js

```diff
@@ -47,11 +47,11 @@ function createLocalizedElement(tagName, valueL10n) {
   } else if (valueL10n.id) {
     navigator.mozL10n.setAttributes(element, valueL10n.id, valueL10n.args);
   // if we passed in a HTML Fragment, it is already localized
-  } else if (valueL10n.raw.nodeType) {
+  } else if (valueL10n.raw && valueL10n.raw.nodeType) {
     element.appendChild(valueL10n.raw);
   // otherwise - stuff text in here...
   } else {
-    element.textContent = valueL10n.raw;
+    element.textContent = valueL10n.raw || '';
```

Comment:
I have a mixed feelings about this line, maybe it's better to leave it as is, otherwise we'll have empty string for _0_ or _false_, maybe not real use case, but would be easier to catch bugs if something wrong is passed here. What do you think?